YellowNotes.Core/Services/UserService.cs

```diff
@@ -63,16 +63,5 @@ public string GenerateJWT(UserDto user)
 
             return tokenHandler.WriteToken(token);
         }
-
-        public bool ValidateToken(string token, UserDto user)
-        {
-            var securityToken = new JwtSecurityTokenHandler().ReadToken(token) as JwtSecurityToken;
-            var decodedEmail = securityToken.Payload["email"] as string;
-
-            var isUserAuthorized = decodedEmail == user.Email;
-            var tokenExpired = securityToken.ValidTo < DateTime.UtcNow;
-
-            return isUserAuthorized && !tokenExpired;
-        }
```

Comment:
This function is necessary. Without this validation it's possible to send one email in body and another encoded in token and api will accept such request while it shouldn't.

Reply:
Alright. I'm gonna bring it back from the dead 😃.